src/Proto3/Wire/Encode.hs

```diff
@@ -122,6 +129,13 @@ instance Show MessageBuilder where
     where
       bytes' = toLazyByteString builder
 
+etaMessageBuilder :: (a -> MessageBuilder) -> (a -> MessageBuilder)
+etaMessageBuilder = (MessageBuilder .) . RB.etaBuildR . (unMessageBuilder .)
+
+ensureMessageBuilder :: Int -> MessageBuilder -> MessageBuilder
+ensureMessageBuilder required =
```

Comment:
Similarly, I think this is just:

```haskell
ensureMessageBuilder = coerce RB.ensure
```

Reply:
OK.

Reply:
I ended up eliminating this function entirely in 5fefcb2713f0e08d0d170cfa09397b314bce16f9 .